pages/linux/ufw.md

```diff
@@ -2,6 +2,7 @@
 
 > Uncomplicated Firewall.
 > Frontend for iptables aiming to make configuration of a firewall easier.
+> More information: <https://wiki.ubuntu.com/UncomplicatedFirewall>.
```

Comment:
Why not <https://manned.org/ufw> @bl-ue?

Reply:
I didn't check either that or the wiki link. 😞

I tihnk the manned.org page is really useful tho, so I guess we should open a new PR to fix it 😂